akka-cluster/src/main/scala/akka/cluster/ClusterSettings.scala

```diff
@@ -105,7 +105,7 @@ final class ClusterSettings(val config: Config, val systemName: String) {
     }
   }
 
-  val PruneGossipTombstonesAfter: Duration = {
+  val PruneGossipTombstonesAfter: FiniteDuration = {
```

Comment:
this is public so probably not worth fighting bin compat

Reply:
I'll add a doc entry that it will always be finite and revert.